driver/InputFlags.hpp

```diff
@@ -57,6 +57,7 @@ class InputFlags
     int GetValueInt(const std::string& _long_name) const;
     uint64_t GetValueUint64(const std::string& _long_name) const;
     double GetValueDouble(const std::string& _long_name) const;
+    void SetValue(const std::string& _long_name, const std::string& _new_value);
```

Comment:
No leading underscores, please. We will definitely face problems, it's only a matter of time.

Reply:
OK